### ISSUE TITLE
Diagram for jasper ?

### ISSUE THREAD

**Issue by hazouga** [X360Tools/PicoRGH#7]
rgh 3 not stable for phat consoles and many recommend do rgh 1.2
but we didn't have these glitch chip in my country  
can any one help me please ?

**Comment by chackAJMCPE** [comment]
I can help. I compiled it (i didnt test)
Pinout: 

GPIO15 - POST
GPIO12 - CPU_PLL_BYPASS
GPIO11 - CPU_RESET
GND - GND
3V3 - 3V3



and download the UF2 here:    https://www.mediafire.com/file/1fizlyf6o8ztuf6/PicoRGH.uf2/file

Tell me if this works.

**Comment by anthonyg45157** [comment]
> I can help. I compiled it (i didnt test) Pinout:
> 
> GPIO15 - POST GPIO12 - CPU_PLL_BYPASS GPIO11 - CPU_RESET GND - GND 3V3 - 3V3
> 
> and download the UF2 here: https://www.mediafire.com/file/1fizlyf6o8ztuf6/PicoRGH.uf2/file
> 
> Tell me if this works.

I have a Jasper on the way and am HIGHLY interested in trying this. Do you have another method of contact so we can discuss this in further detail TY

**Comment by patricksouza472** [comment]
> I can help. I compiled it (i didnt test) Pinout:
> 
> GPIO15 - POST GPIO12 - CPU_PLL_BYPASS GPIO11 - CPU_RESET GND - GND 3V3 - 3V3
> 
> and download the UF2 here: https://www.mediafire.com/file/1fizlyf6o8ztuf6/PicoRGH.uf2/file
> 
> Tell me if this works.

Oh boy I'm gonna test it and report back

**Comment by anthonyg45157** [comment]
> > I can help. I compiled it (i didnt test) Pinout:
> > GPIO15 - POST GPIO12 - CPU_PLL_BYPASS GPIO11 - CPU_RESET GND - GND 3V3 - 3V3
> > and download the UF2 here: https://www.mediafire.com/file/1fizlyf6o8ztuf6/PicoRGH.uf2/file
> > Tell me if this works.
> 
> Oh boy I'm gonna test it and report back

Looking forward to the report back!!

**Comment by patricksouza472** [comment]
> I can help. I compiled it (i didnt test) Pinout:
> 
> GPIO15 - POST GPIO12 - CPU_PLL_BYPASS GPIO11 - CPU_RESET GND - GND 3V3 - 3V3
> 
> and download the UF2 here: https://www.mediafire.com/file/1fizlyf6o8ztuf6/PicoRGH.uf2/file
> 
> Tell me if this works.

It didn't work

**Comment by patricksouza472** [comment]
> I can help. I compiled it (i didnt test) Pinout:
> 
> GPIO15 - POST GPIO12 - CPU_PLL_BYPASS GPIO11 - CPU_RESET GND - GND 3V3 - 3V3
> 
> and download the UF2 here: https://www.mediafire.com/file/1fizlyf6o8ztuf6/PicoRGH.uf2/file
> 
> Tell me if this works.

It didn't work

**Comment by chackAJMCPE** [comment]
It looks like the timing is for trinity.

**Comment by aayes89** [comment]
I will test a Xenon with the latest firmware available before the store goes down and we will see the results when it is finished.

**Comment by chackAJMCPE** [comment]
> I will test a Xenon with the latest firmware available before the store goes down and we will see the results when it is finished.

If you have dash 7371 or below DO NOT UPDATE.
You can do a better hack. The JTAG hack. It boots in seconds (unlike RGH that boots unreliable and is hard to get timings right with a glitch chip) and it doesn't require very expensive stuff to use. Two diodes and some cables.

**Comment by chackAJMCPE** [comment]
@patricksouza472 
I will try to get some timings right and maybe there 2ill be success on phat xboxes.

**Comment by aayes89** [comment]
> > I will test a Xenon with the latest firmware available before the store goes down and we will see the results when it is finished.
> 
> If you have dash 7371 or below DO NOT UPDATE.
> You can do a better hack. The JTAG hack. It boots in seconds (unlike RGH that boots unreliable and is hard to get timings right with a glitch chip) and it doesn't require very expensive stuff to use. Two diodes and some cables.

No, I'm not. The latest version available: 2.0.17559.0 without the Microsoft's shutdown of the marketplace and some other applications censorship on the dashboard. I'm actually running without internet using a local router connection

**Comment by anthonyg45157** [comment]
I have a Jasper I would love to try this on if anyone has ideas on how to compile a timing file for that

**Comment by Meska12** [comment]
> [@patricksouza472](https://github.com/patricksouza472) I will try to get some timings right and maybe there 2ill be success on phat xboxes.

Did you try get right timings? And I think we have to use 22k resistor on PLL like we do on ACE glitcher

**Comment by sl1pkn07** [comment]
also interested on jasper timmings

as i knowed recently, may be need 1.2v regulator for PLL and POST due the 3.3v of the pico, like when you do the RGH3

**Comment by chackAJMCPE** [comment]
> > [@patricksouza472](https://github.com/patricksouza472) I will try to get some timings right and maybe there 2ill be success on phat xboxes.
> 
> Did you try get right timings? And I think we have to use 22k resistor on PLL like we do on ACE glitcher

I forgot about this all, but i'm maybe going to do it this month
And you should use a resistor. i dont think 22k but something i would say 7k-17k, if resistance is lower, then chances of glitching are better, but more risk of damaging the cpu

As for timings, i have a simple solution: count clock cycles until the final post code, calculate 62% of the whole length, reboot and use that. This should make it universal for most of the xboxes. But there is a problem. Phats have slowdown factor of 128x, where slims have 640x. The pico would need to be much faster, or have worse reliability rate. I dont know if this is possible, its already overclocked to 266mhz. But i could port this to coronas for sure

Nevermind, this is dumb, idk why its even overclocked.

**Comment by chackAJMCPE** [comment]
Nevermind. This would achieve nothing more than what rgh3 does on phats, and i'm not going to port it to just 1 other console.
